docs/content/gds/admin/configuration/gds.en.md

```diff
@@ -1,7 +1,159 @@
+
+### Email and SendGrid
+
+The GDS uses [SendGrid](https://sendgrid.com/) to send email notifications and to enable communication workflows with users and admins. Configure GDS to use SendGrid as follows:
+
+| EnvVar                 | Type   | Default                                | Description                                                                                 |
+|------------------------|--------|----------------------------------------|---------------------------------------------------------------------------------------------|
+| GDS_SERVICE_EMAIL      | string | TRISA Directory Service                | The email address used as the sender for all emails from the GDS system.                    |
+| GDS_ADMIN_EMAIL        | string | TRISA Admins                           | The email address to send admin emails to from the server.                                  |
+| SENDGRID_API_KEY       | string |                                        | API Key to authenticate to SendGrid with.                                                   |
+| GDS_DIRECTORY_ID       | string | vaspdirectory.net                      | (Reused) The network ID the GDS serves, either vaspdirectory.net or trisatest.net (or .dev) |
+| GDS_VERIFY_CONTACT_URL | string | https://vaspdirectory.net/verify       | The base URL to include in emails for contact verification                                  |
+| GDS_ADMIN_REVIEW_URL   | string | https://admin.vaspdirectory.net/vasps/ | The base URL to include in emails to link to a new VASP registration                        |
+| GDS_EMAIL_TESTING      | bool   | false                                  | Use email in testing mode rather than send live emails                                      |
+| GDS_EMAIL_STORAGE      | string | ""                                     | Directory to store test emails for "mark one eyeball" review                                |
+
+SendGrid is considered **enabled** if the SendGrid API Key is set. The service and admin email addresses are required if SendGrid is enabled.
+
+### CertMan
+
+The CertMan is a background process that runs on the GDS and manages certificate requests, interacts with Sectigo, and finalizes the VASP registration process. It is configured as follows:
+
+| EnvVar                          | Type     | Default           | Description                                                                                                    |
+|---------------------------------|----------|-------------------|----------------------------------------------------------------------------------------------------------------|
+| GDS_CERTMAN_ENABLED             | bool     | true              | If false, disables the CertMan background process.                                                             |
```

Comment:
The description sentence here ends in a `.` while others don't. This is a consistency nitpick and may be ignored.